cmd/object-handlers.go

```diff
@@ -1352,6 +1352,10 @@ func (api objectAPIHandlers) CopyObjectHandler(w http.ResponseWriter, r *http.Re
 	// Write success response.
 	writeSuccessResponseXML(w, encodedSuccessResponse)
 
+	// avoid generating a notification for REPLICA creation event.
+	if _, ok := r.Header[xhttp.MinIOSourceReplicationRequest]; ok {
+		return
+	}
 	// Notify object created event.
```

Comment:
Can this be directly added inside `sendEvent()` in one place? - you should also `extractReqParams(r)` for this to be ingored in sendEvent()

Reply:
@poornas ^^

Reply:
that's a good idea, done.